addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -42,5 +43,29 @@ tour.register('edit_megamenu', {
         trigger: '#top_menu .nav-item a.o_mega_menu_toggle:contains("Megaaaaa!")',
         run: function () {}, // it's a check
     },
+    // Edit a menu item
+    wTourUtils.clickOnEdit(),
+    {
+        content: "Toggle the mega menu",
+        trigger: '#top_menu .o_mega_menu_toggle'
+    },
+    {
+        content: "Click on the first title item",
+        trigger: '.o_mega_menu h4',
+    },
+    {
+        content: "Press enter",
+        trigger: '.o_mega_menu h4',
+        run: function (actions) {
+            this.$anchor[0].dispatchEvent(new InputEvent('input', {bubbles: true, inputType: 'insertParagraph'}));
+            actions.text('New Menu Item', this.$anchor[0]);
+        },
+    },
+    {
+        content: "Menu Item should have been renamed",
+        trigger: '.o_mega_menu h4:contains("New Menu Item")',
+        run: function () {}, // it's a check
+    },
+    ...wTourUtils.clickOnSave(),
```

Comment:
Cool to complete this test but makes me wonder how you tested it... as I don't see it being run by a python test :grimacing: The commit which [introduces it ](https://github.com/odoo/odoo/commit/fef5c6eff78101d3903f3e8f2a40b92dc7dc9d7b) does not start it in any way... maybe check with them? :cry:

Reply:
I was playing it in `?debug=tests` from the console with `odoo.startTour('edit_megamenu')` but indeed it was never played from python with the other at_install tests...